src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1,3 +1,20 @@
-export const CommentInfo = () => (
-  <>Put the comment here</>
+export const CommentInfo = ({ comment }) => (
+  <div className="CommentInfo">
+    <div className="CommentInfo__title">
+      <strong className="CommentInfo__name">{comment.name}</strong>
+
+      {' by '}
```

Comment:
There is unnecessary space between the text and the component. The space can be combined with the text in the previous or next line.